dataeng/resources/opsgenie-enable-heartbeat.sh

```diff
@@ -25,7 +25,23 @@ if [ -n "$OPSGENIE_HEARTBEAT_NAME" ] && \
     HEARTBEAT_API_PING_URL="$HEARTBEAT_API_URL/ping"
     ALERT_MESSAGE="Heartbeat [$OPSGENIE_HEARTBEAT_NAME] expired - job is likely stuck."
     JSON_REQ_DATA="{ \"interval\": \"$OPSGENIE_HEARTBEAT_DURATION_NUM\", \"intervalUnit\": \"$OPSGENIE_HEARTBEAT_DURATION_UNIT\", \"alertMessage\": \"$ALERT_MESSAGE\", \"enabled\": true }"
-
+    
+    # Make API request to get existing heartbeat
+    # If the heartbeat already exist, then GET_EXISTING_HEARTBEAT will set to heartbeat name "GET_EXISTING_HEARTBEAT=heatbeat_name"
+    # If heartbeat don't exist GET_EXISTING_HEARTBEAT is set to empty
+    GET_EXISTING_HEARTBEAT=$(curl -X GET $HEARTBEAT_API_URL --header "$AUTH_HEADER" | grep -o $OPSGENIE_HEARTBEAT_NAME | sort -u)
```

Comment:
is `| sort -u` essential for functionality, or can it be removed?

Reply:
The json response has heartbeat name more than one time, if I do grep -o heartbeat_name it returns duplicates so it **sort -u** de-depulicate and returns heartbeat name only once so I think its better to keep it there